### PR TITLE
Introducing high performant endpoint analysis to perform stack analysis

### PR DESCRIPTION
**Pull request by samuzzal-choudhury**

### CODE REVIEW

File: bayesian/api_v1.py [hunk 1]
```diff
@@ -840,7 +845,116 @@ def post():
...
+            api_url = current_app.config['F8_API_BACKBONE_HOST']
+
+            d = DependencyFinder()
+            deps = d.execute(args, rdb.session)
+            deps['external_request_id'] = request_id
+
+            _session.post('{}/api/v1/stack_aggregator'.format(api_url), json=deps)
+            _session.post('{}/api/v1/recommender'.format(api_url), json=deps)
+        except Exception as exc:
+            print(exc)
```
Comment (miteshvp) [inline bayesian/api_v1.py]: remove print debugs

File: hack/coreapi-server.sh [hunk 2]
```diff
@@ -12,13 +12,15 @@ exec mod_wsgi-express start-server \
                       --port 5000 \
                       --user coreapi \
                       --group coreapi \
-                      --processes 4 \
-                      --threads 10 \
+                      --processes 1 \
```
Comment (miteshvp) [inline hack/coreapi-server.sh]: Let's try load testing with multi-process multi-threads here.

Comment (samuzzal-choudhury) [reply hack/coreapi-server.sh]: @miteshvp need to try and check. Mercator CLI runs on a thread and having a thread to run another thread basically halts the processing and results in gateway timeout. I will do my research and get back on this.

Comment (samuzzal-choudhury) [reply hack/coreapi-server.sh]: @miteshvp in order to run stack analysis with the verified performance results, we need this configuration in coreapi-server.sh

**Comment by miteshvp** [comment]
LGTM.

**Pull request merged**